CONTRIBUTING.md

```diff
@@ -502,6 +510,45 @@ kubectl -n crossplane-system scale deploy crossplane --replicas=0
 make run
 ```
 
+## Code Review Process
+
+All Pull Requests (PR), whether written by a Crossplane maintainer or a
+community member, must go through code review. This ensures that the code is
+correct, maintainable, and secure.
+
+We encourage anyone in the community to conduct a code review on a PR,
+however, we require the following approvals before merging a PR:
+- At least one approval from [Reviewers] or [Code Owners]
+- At least one approval from [Maintainers]
+
+When opening a PR, GitHub will automatically assign the
+[crossplane-reviewers] and [crossplane-maintainers] teams, as well as
+any [Code Owners]. The PR author may also add any additional reviewers they
+might want to have a look at the PR.
```

Comment:
My intention was that in most cases (depending on the path in CODEOWNERS):

* Two reviewers are automatically assigned from the crossplane-reviewers group.
* One, specific maintainer who is an SME for that part of the codebase is assigned.

See for example:

* https://github.com/crossplane/crossplane/blob/377239ea/CODEOWNERS#L39, which assigns reviewers and Dan for all things package manager
* https://github.com/crossplane/crossplane/blob/377239ea/CODEOWNERS#L45, which assigns reviewers and Muvaf for all things Composition.

I was confused at first by the fact that crossplane-maintainers _seems_ to be being assigned to most PRs, but after doing a bit of poking around I think this is just GitHub behaving in a slightly misleading way. I'm pretty sure it's always assigning the specific code owner (who by convention always happens to be a member of crossplane-maintainers) but also adding crossplane-maintainers to indicate that the code owner represents that group?

See for example:

* https://github.com/crossplane/crossplane/pull/3364 - Assigned muvaf (Composition SME)
* https://github.com/crossplane/crossplane/pull/3450 - Assigned muvaf (Composition SME)
* https://github.com/crossplane/crossplane/pull/3445 - Assigned muvaf (Composition SME)
* https://github.com/crossplane/crossplane/pull/3444 - Assigned me (RBAC manager SME)

Reply:
So all that said, I'd suggest perhaps point out:

* That who gets assigned depends on what code the contributor changed, per CODEOWNERS.
* That in _most_ cases we expect crossplane-reviewers and a subject matter expert maintainer to be assigned.